src/transformation/visitors/binary-expression/compound.ts

```diff
@@ -48,7 +48,7 @@ type CompoundAssignmentToken =
     | ts.SyntaxKind.GreaterThanGreaterThanToken
     | ts.SyntaxKind.GreaterThanGreaterThanGreaterThanToken;
 
-const compoundToAssignmentTokens: Record<ts.CompoundAssignmentOperator, CompoundAssignmentToken> = {
```

Comment:
Which CompoundAssignmentOperator do we not support now?

Reply:
??=, &&=, ||=

Reply:
For this PR we should either create an issue to support these compound operators later, or just revert the Partial/ | undefined changes and add them here, but then we also need some tests for them. I'm leaving it up to @hazzard993 to decide.

Reply:
Created #916 to complete this later